xscen/data/file_schema.yml

```diff
@@ -0,0 +1,192 @@
+#               # The special "DATES" facet will be replaced by the most concise way found to define the temporal range covered by the file.
+---
+### Raw data
+#
+# These schemas are meant for raw-like data, usually hourly or daily, used as input for scenario projects.
+# It includes bias-adjusted datasets.
+raw-non-sims:
+  with:
+    - option: type
+      value: [station-obs, reconstruction, forecast]
+    - option: processing_level
+      value: raw
+  folders:
+    - type
+    - domain
+    - institution
+    - [source, version]
+    - member
+    - frequency
+    - variable
+  filename: [ variable, frequency, domain, institution, source, version, member, DATES ]
+raw-sims:
+  with:
+    - option: type
+      value: simulation
+    - option: processing_level
+      value: [ raw, biasadjusted ]
+  folders:
+    - type
+    - processing_level
+    - option: bias_adjust_project
+      is_true: [bias_adjust_project, version]
+    - mip_era
+    - activity
+    - domain
+    - institution
+    - source
+    - driving_model
+    - experiment
+    - member
+    - frequency
+    - variable
+  filename: [ variable, frequency, bias_adjust_project, version, mip_era, activity, domain, institution, source, driving_model, experiment, member, DATES ]
+raw-hydro-reconstruction:
+  with:
+    - option: type
+      value: reconstruction-hydro
+    - option: processing_level
+      value: raw
+  folders:
+    - type
+    - domain
+    - hydrology_project
+    - hydrology_institution
+    - hydrology_source
+    - institution
+    - [source, version]
+    - member
+    - frequency
+    - variable
+  filename: [ variable, frequency, domain, hydrology_project, hydrology_institution, hydrology_source, institution, source, version, member, DATES ]
+raw-hydro-sims:
+  with:
+    - option: type
+      value: simulation-hydro
+    - option: processing_level
+      value: [ raw, biasadjusted ]
+  folders:
+    - type
+    - hydrology_project
+    - hydrology_institution
+    - hydrology_source
+    - processing_level
+    - option: bias_adjust_project
+      is_true: [bias_adjust_project, version]
+    - mip_era
+    - activity
+    - domain
+    - institution
+    - source
+    - driving_model
+    - experiment
+    - member
+    - frequency
+    - variable
+  filename: [ variable, frequency, hydrology_project, hydrology_institution, hydrology_source,  bias_adjust_project, version, mip_era, activity, domain, institution, source, driving_model, experiment, member, DATES ]
+### Derived data
+#
+# These schemas are meant for the output of scenario projects.
+# The main differences here are that:
+#
+#     - domain denotes the zone over which the scenario was computed, not the original source's coverage
+#     - processing_level has more possible values, denoting slight variant of the data instead of broad categories.
+#     As such, both fields are here much lower in the hierarchy.
+derived-sims:
+  with:
+  - option: type
+    value: simulation
+  folders:
+    - type
```

Comment:
Where will does go ? If we just put them in /datasets like that, we will have:
|_ simulation
            |_ raw
            |_ biasadjusted
            |_ ESPO-G6-R2_v1.0 #containing indicators and other derived...

If it stays in /datasets, we could add "derived" after type? 
Or we just know that we have to put the root somewhere else ?